xbmc/platform/win32/DisplayUtilsWin32.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "HDRStatus.h"
+
+#include <optional>
+#include <string>
+#include <vector>
+
+#include <dxgi1_5.h>
+
+class CDisplayUtilsWin32
+{
+public:
+  CDisplayUtilsWin32() = delete;
```

Comment:
```suggestion
  CDisplayUtilsWin32() = default;
```

Reply:
why? we don't ever want to construct one. Same remark for the destructor

Reply:
You're right, delete is fine if not used.